setup.py

```diff
@@ -1,16 +1,7 @@
 #!/usr/bin/env python
 # encoding: utf-8
 
-# n.b. we can't have unicode_literals here due to http://bugs.python.org/setuptools/issue152
-from __future__ import absolute_import, division, print_function
-
-try:
-    from setuptools import setup
```

Comment:
This seems fine since we no longer support Python 2.x old enough not to have `setuptools` available.